cunumeric/random/bitgenerator.py

```diff
@@ -12,20 +12,29 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
+from __future__ import annotations
+
+from typing import TYPE_CHECKING, Union
 
 import numpy as np
-from cunumeric.array import ndarray
-from cunumeric.config import BitGeneratorType
-from cunumeric.runtime import runtime
+
+from ..array import ndarray
+from ..config import BitGeneratorType
+from ..runtime import runtime
+
+if TYPE_CHECKING:
+    import numpy.typing as npt
+
+    from ..types import NdShapeLike
 
 
 class BitGenerator:
     def __init__(
         self,
-        seed=None,
-        generatorType=BitGeneratorType.DEFAULT,
-        forceBuild=False,
-    ):
+        seed: Union[int, None] = None,
```

Comment:
Should this be `seed: int = 0` instead? the `seed` value feeds into `task.add_scalar_arg(seed, ty.uint64)` is None acceptable to pass there?

Reply:
That's the reference
https://numpy.org/doc/stable/reference/random/bit_generators/generated/numpy.random.BitGenerator.html

However, in the current implementation, we only support int as a parameter, or None (which in our case is zero but could be changed)

Reply:
@manopapad after discussion with @fduguet-nv it seems like it's not clear what to do with None values, presently. Should we raise an explicit error similar to other places where we can't handle certain kinds of values?

Reply:
We could also just make `seed` a required argument, which diverges from numpy slightly, but has the same practical effect (just a different exception raised)

Reply:
If seed is none, could we set seed to `time.perf_counter_ns()` ? I think that's the most meaningful thing to do here

Reply:
:+1: that is certainly simple to do, I will push the change. Normally I would also add a unit test with a mock to ensure that `time.perf_counter_ns` is actually called in this case, but I am not sure that will be simple with our oddball testing situation

Reply:
https://github.com/nv-legate/cunumeric/pull/471/commits/eb1c8649d5007c933afd5649d943ea0a9c13dc3e